plugins/CoreConsole/Commands/DevelopmentSyncUITestScreenshots.php

```diff
@@ -11,15 +11,21 @@
 
 use Piwik\Development;
 use Piwik\Http;
+use Piwik\Log;
 use Piwik\Plugin\ConsoleCommand;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
+ * Tool for core developers to help making UI screenshot builds green.
+ *
  */
 class DevelopmentSyncUITestScreenshots extends ConsoleCommand
 {
+    protected $urlBase;
```

Comment:
This is a pattern I've been trying to avoid recently: storing data that's computed in execute() in member variables. When I do it, I find the code can get confusing, since it's usually expected for state to be initialized upon construction. I think in the end, it's not a big deal, but you might want to compute this in execute() and pass it to necessary methods. If it improves clarity.